FindYourRecipe/FindYourRecipe.Api/Controllers/CategoryRecipeController.cs

```diff
@@ -32,14 +48,21 @@ public async Task<IActionResult> DeleteAsync(int id)
 			try
 			{
 				await CategoryRecipeService.DeleteAsync(id);
-				return Ok();
+				return NoContent();
 			}
 			catch (NotFoundException)
 			{
 				return NotFound();
 			}
-				
 		}
-	}
+
+        [HttpDelete("by-recipeId/{recipeId}")]
+        public async Task<IActionResult> DeleteByRecipeIdAsync(int recipeId)
```

Comment:
Please make sure that the format of all the files will improve as now the indentation is making the process of reading code quite complicated. (`CTRL` + 'K`  + `D` should help)